sympy/calculus/util.py

```diff
@@ -10,6 +10,131 @@
 from sympy.functions.elementary.miscellaneous import Min, Max
 
 
+def continuous_in(f, symbol, interval):
+    """
+    Finds the intervals of continuity of a function in a given interval range.
```

Comment:
@kshitij10496 I agree with @smichr,  `domain` argument doesn't mean the domain of the function, its rather, the domain in which the user is interested to know the continuity of the function `f`. Therefore the `continuous_domain` seems more apt name for this function and `domain` being its argument.